### PR TITLE
Proposing a means to embed learning rate finder (lr_find) before fitting multivariate models

### PR DESCRIPTION
**Pull request by carusyte**
idea: https://github.com/Nixtla/neuralforecast/discussions/1262

### CODE REVIEW

**Comment by elephaint** [comment]
Thanks for your contribution;

Please follow [our contributing guide](https://github.com/Nixtla/neuralforecast/blob/main/CONTRIBUTING.md); we use `nbdev` so direct changes to .py files are generally not allowed. 

I like the idea as it seems simple and straightforward, can you provide a piece of code that showcases the functionality?

**Comment by marcopeix** [comment]
No need of this PR, we can do:
```python
from pytorch_lightning.callbacks import LearningRateFinder
models = [NHITS(h=12, input_size=24, max_steps=10, learning_rate=1e-3, callbacks=[LearningRateFinder()])]
```
